### DISCUSSION TITLE
Stop verdaccio server

### DISCUSSION THREAD

**Discussion by csimpi** [verdaccio/verdaccio#2331] [Q&A]
**Is your feature request related to a problem?**

I would like to run verdaccio server in the background with a shell script. I didn't find any parameter to put it in the background and be silent, so I just run `sh -c "nohup verdaccio &>$VERDACCIO_LOG_FILE &"`. 

The problem is, I can't figure out how to stop the server.
`sudo killall verdaccio` doesn't work

**Describe the solution you'd like**

Any command-line solution to stop verdaccio server

**Describe alternatives you've considered**

using killall, checking top and ps ax to find what process should I stop, but I didn't find anything about verdaccio

**Additional context**

**Comment by mbtools** [comment]
There are multiple options: check [Server Configuration](https://verdaccio.org/docs/server-configuration)
